setup.py

```
@@ -0,0 +1,27 @@
+Package setup configuration for grok-lab
+"""
+
+from setuptools import setup, find_packages
+
+setup(
+  name="grok-lab",
+  version="0.1.0",
+  author="Dmitri Zaitsev",
+  author_email="dmitri14@gmail.com",
+  description="A client library for interacting with Grok-3 API",
+  long_description=open("README.md").read(),
```

Ruff (0.11.9)

Use a context manager for opening files

(SIM115)